src/test/resources/testing-editor.md

```diff
@@ -134,10 +139,15 @@
 **Error notification appears when pasting invalid content**
 1. "Edit..." resource
 2. paste invalid yaml
+Bugs:
+1. some weird error message
```

Comment:
weird? Is this what one can see in the screenshots? I'd need the yaml to reproduce this please.

Reply:
could replicate it, fixed it in https://github.com/redhat-developer/intellij-kubernetes/pull/178/commits/6be9eadad87f5136ee87a3d2f498588a83815b68

Reply:
@olkornii confirmed that it now works for him: 
![image](https://user-images.githubusercontent.com/25126/121268241-d7cf6700-c8bd-11eb-848d-ac4bb2820b54.png)
Resolving this conversation.